README.md

```diff
@@ -50,6 +50,7 @@ Store is used to map an upload's fingerprint with the corresponding upload URL.
 | Name | Backend | Dependencies |
 |:----:|:-------:|:------------:|
 | MemoryStore | In-Memory | None |
+| LeveldbStore | LevelDB | None |
```

Comment:
Depends on what you mean with "Dependencies", it won't require an external database such as Redis etc would. This is what I thought you meant with Dependencies.